android/src/main/java/com/tanguyantoine/react/MusicControlNotification.java

```diff
@@ -50,21 +52,26 @@ public void show(NotificationCompat.Builder builder, boolean isPlaying) {
         if(stop != null) builder.addAction(stop);
         if(next != null) builder.addAction(next);
 
-        // Notifications of playing music can't be removed
-        builder.setOngoing(isPlaying);
+        // Set whether notification can be closed based on closeNotification control (default PAUSED)
+        if(module.notificationClose == MusicControlModule.NotificationClose.ALWAYS) {
+            builder.setOngoing(false);
+        } else if(module.notificationClose == MusicControlModule.NotificationClose.PAUSED) {
+            builder.setOngoing(isPlaying);
+        } else { // NotificationClose.NEVER
+            builder.setOngoing(true); 
+        }
+
         builder.setSmallIcon(smallIcon);
 
         // Open the app when the notification is clicked
         String packageName = context.getPackageName();
         Intent openApp = context.getPackageManager().getLaunchIntentForPackage(packageName);
         builder.setContentIntent(PendingIntent.getActivity(context, 0, openApp, 0));
 
-        if(!isPlaying) {
-            // Remove notification
-            Intent remove = new Intent(REMOVE_NOTIFICATION);
-            remove.putExtra(PACKAGE_NAME, context.getApplicationInfo().packageName);
-            builder.setDeleteIntent(PendingIntent.getBroadcast(context, 0, remove, PendingIntent.FLAG_UPDATE_CURRENT));
-        }
+        // Remove notification
+        Intent remove = new Intent(REMOVE_NOTIFICATION);
+        remove.putExtra(PACKAGE_NAME, context.getApplicationInfo().packageName);
+        builder.setDeleteIntent(PendingIntent.getBroadcast(context, 0, remove, PendingIntent.FLAG_UPDATE_CURRENT));
```

Comment:
If this intent is not conditional anymore, you should move it to the constructor

Reply:
I could do that, but the builder object is local to this method, so I would have to store the intent off as a class variable.  If I were to do that, it would make sense to do the same to the openApp intent, which was not conditional originally but is still constructed in this method.